src/bisect.cpp

```diff
@@ -14,13 +15,23 @@
 #include "options.hpp"
 #include "util/interval.hpp"
 
+#include <boost/functional/hash.hpp>
+#include <boost/unordered_map.hpp>
+
 namespace parameters {
 constexpr long stopping_time_par = 3;
 }
 constexpr bool print_time = true;
 
 namespace pmondriaan {
 
+// This is the function used for hashing the vectors, may be very inefficient.

```

Comment:
Is this used anywhere or was this for testing purposes?

Reply:
I believe this is used for efficient hashmaps.